### PR TITLE
Enable black

### PR DESCRIPTION
**Pull request by acmnu**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+[tool.black]
+line-length = 100
+skip-string-normalization = true
+target-version = ['py38']
```
Comment (dgusakov) [inline pyproject.toml:4]: But we use 3.9

**Review by dgusakov — APPROVED** [review]

**Review by nagarin — DISMISSED** [review]
6000 lines are hardly possible to read properly

**Review by nagarin — DISMISSED** [review]

**Review by a-alferov — APPROVED** [review]

**Comment by acmnu** [comment]
bender build

**Comment by BenderArenadata** [comment]
Hey, @acmnu! Build has been launched!

**Review by grey-kristy — APPROVED** [review]

**Pull request merged**